pkgs/by-name/ho/homer/package.nix

```diff
@@ -0,0 +1,62 @@
+{
+  lib,
+  fetchFromGitHub,
+  pnpm,
+  nodejs,
+  pkgs,
+}:
+pkgs.stdenv.mkDerivation rec {
+  pname = "homer";
+  version = "24.11.1";
+  src = fetchFromGitHub {
+    owner = "bastienwirtz";
+    repo = "homer";
+    rev = "v${version}";
+    hash = "sha256-a2t8qd/yxk7R0zyhQo0zoSIkfzfvBIrzNclNsD5GxRw=";
+  };
+
+  pnpmDeps = pnpm.fetchDeps {
+    inherit
+      pname
+      version
+      src
+      patches
+      ;
+    hash = "sha256-5unoY8lPaX9sZAJEBICpxSddwLV8liK1tbamB2ulvew=";
+  };
+
+  patches = [ ./sass-embedded.patch ];
+
+  nativeBuildInputs = with pkgs; [
```

Comment:
`with pkgs` shouldn't be used as it's unintuitive in regards to which package is in scope.